CorsixTH/Lua/room.lua

```diff
@@ -258,10 +258,10 @@ function Room:testStaffCriteria(criteria, extra_humanoid)
     end
     return false
   else
-    for _, _ in pairs(missing) do
-      return false
+    if next(missing) == nil then
+      return true
     end
-    return true
+    return false
```

Comment:
`return next(missing) == nil` just like the previous change?

Reply:
Sure